test/tests/can.c

```diff
@@ -152,7 +187,10 @@ int main(void) {
   hcan1->Init.AutoWakeUp = DISABLE;
   hcan1->Init.AutoRetransmission = ENABLE;
   hcan1->Init.ReceiveFifoLocked = DISABLE;
-  hcan1->Init.TransmitFifoPriority = DISABLE;
+
+  // To guarantee order, this must be enabled.
+  // Disable if order of transmission does not matter.
```

Comment:
Do not just say this. Explain what is happening. If it is disabled, the hardware chooses from the three mailboxes based on the ID. If enabled, the hardware acts like a FIFO and chooses based on time.